runtime/lua/vim/lsp/tagfunc.lua

```diff
@@ -0,0 +1,74 @@
+local lsp = vim.lsp
+local util = vim.lsp.util
+
+local function mk_tag_item(name, range, uri, offset_encoding)
+  local bufnr = vim.uri_to_bufnr(uri)
+  -- This is get_line_byte_from_position is 1-indexed, call cursor expects a 0-indexed byte
+  local byte = util._get_line_byte_from_position(bufnr, range.start, offset_encoding) - 1
```

Comment:
```suggestion
  -- This is get_line_byte_from_position converts to 0-index, cursor is 1 indexed
  local byte = util._get_line_byte_from_position(bufnr, range.start, offset_encoding) + 1
```

Now it is off by two :)

From the `get_line_byte_from_position` docs:

> Returns a zero-indexed column, since set_lines() does the conversion to
> 1-indexed

From the `cursor()` docs:

> Positions the cursor at the column (byte count) {col} in the
> line {lnum}.  **The first column is one**.

Reply:
🤦

Reply:
fixed I think